extensions/EditComments/Extension.pm

```diff
@@ -63,6 +63,13 @@ sub install_update_db {
   # because existing admin-edited revisions may contain sensitive info
   $dbh->bz_add_column('longdescs_activity', 'is_hidden',
     {TYPE => 'BOOLEAN', NOTNULL => 1, DEFAULT => 1});
+
```

Comment:
Does this impact search? I can't remember what happens for fielddefs that aren't also defined in the search code.

Reply:
Just checked... Comment Revisions appears in the Custom Search dropdown, which isn’t necessary. I’ll remove this field definition. It still works; it will just say “Comment X” instead of “Comment X Revisions”.